helm/superset/templates/deployment.yaml

```diff
@@ -79,6 +79,12 @@ spec:
               value: {{ $value | quote }}
             {{- end }}
           {{- end }}
+          {{- if .Values.supersetNode.env }}
```

Comment:
is this if necessary?

Reply:
@dpgaspar No but after thinking about it I would prefer to keep it to match how extraEnv is read in this file and others.